test/circle2d.jl

```diff
@@ -28,6 +28,29 @@ using LinearAlgebra
         expected = map(i->(i==1 ? Gray{N0f8}(0.5) : Gray{N0f8}(0)), expected)
         @test all(expected .== res) == true
 
+        img = zeros(Gray{N0f8}, 10, 10)
```

Comment:
For each type, we need one more test for the default value, e.g., 

```julia
draw(img, CirclePointRadius(Point(6, 6), 5; thickness = 0, fill = true)) == draw(img, CirclePointRadius(Point(6, 6), 5))
```

This looks tedious but it is an important test case.